tests/test_utils.py

```diff
@@ -386,3 +386,20 @@ def test_deduplicate_and_sort_reqs():
 )
 def test_sql_to_python_return_type_mapper(argument: Tuple[str, str]):
     assert utils.sql_to_python_return_type_mapper(argument[0]) == argument[1]
+
+
+@mock.patch("platform.system")
+@pytest.mark.parametrize(
+    "argument, expected",
+    [
+        ("C:\Something\Something Else", "C:\Something\Something Else"),
+        (
+            "/var/folders/k8/3sdqh3nn4gg7lpr5fz0fjlqw0000gn/T/tmpja15jymq",
+            "/var/folders/k8/3sdqh3nn4gg7lpr5fz0fjlqw0000gn/T/tmpja15jymq",
+        ),
+    ],
+)
+def test_path_resolver(mock_system, argument, expected):
```

Comment:
This test does not check, if the solution works - as it would need a valid win 8.3 path. It just checks if nothing gets broken, and are the normal paths left unchanged.